src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,19 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.services.DriverService;
+import mate.jdbc.services.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        System.out.println(driverService.getAll());
+        System.out.println(manufacturerService.getAll());
```

Comment:
I cannot see usage of create/update/delete operations to check if they are working